### PR TITLE
WebEve: Improvements in REveDataTable

### PR DESCRIPTION
**Pull request by alja**
* fixes in layout to remove unnecessary scrollbars
* implement tabular complete, show suggestions starting with "i." syntax
* provide public method list for tab complete of current and derived classes
* look for predefined column definition in current and base classes

### CODE REVIEW

**Comment by alja** [comment]
@linev Please have a look.

File: ui5/eve7/controller/EveTable.controller.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ sap.ui.define([
    'sap/m/Button',
    "sap/ui/core/ResizeHandler",
    "sap/ui/layout/VerticalLayout",
-   "sap/ui/layout/HorizontalLayout"
+   "sap/ui/layout/HorizontalLayout",
+   "sap/m/MessageBox"
 ], function (Controller, JSONModel, CheckBox, Menu, MenuItem, coreItem, Column,
              mInput, mButton, ResizeHandler, VerticalLayout, HorizontalLayout) {
```
Comment (linev) [inline ui5/eve7/controller/EveTable.controller.js]: Add ",MessageBox" after HorizontalLayout

File: ui5/eve7/controller/EveTable.controller.js [hunk 2]
```diff
@@ -197,61 +198,118 @@ sap.ui.define([
...
+               var collection = this.mgr.GetElement(this.eveTable.fCollectionId);               
+               var exprIn = new sap.m.Input("inputExp", {
+    		  placeholder: "Start expression with \"i.\" to access object",
+    		  showValueHelp: true,
+    		  showTableSuggestionValueHelp: false,
+    		  width: "100%",
+    		  maxSuggestionWidth: "500px",
+    		  showSuggestion: true,
+    		  valueHelpRequest: function (oEvent) {
+    		     sap.m.MessageBox.alert("Write any valid expression.\n Using \"i.\" convetion to access an object in collection. Below is an example:\ni.GetPdgCode() + 2");
```
Comment (linev) [inline ui5/eve7/controller/EveTable.controller.js]: Use MessageBox instead of sap.m.MessageBox

Comment (alja) [reply ui5/eve7/controller/EveTable.controller.js]: I have made the correction. Thanks!

**Pull request merged**